script/Ciel/Day 14/14_01B_02_CIEL14B_2.txt

```diff
@@ -474,25 +474,30 @@ The earth's surface has been stored away,
 [sha:903e70ee68dc77cbb26e8eecf9a6c3c706017a84]{
 -- Page 30, Offset 39428.
 -- 地中から立ち上がる光の巨人。
-The titan of light rises from the ground.
+The giant of light rises from the ground.
 }
 [sha:c7d778846bb42775fe3ae29c24303bd1cfcb9505]{
 -- Page 30, Offset 39429.
 -- ソレは真祖という<規|カ><格|ラ>を見失い、<顕現|そんざい>方法が暴走しただけの力の具現であり―――
 // Maybe this should link with 39399?
-It is no more than an incarnation of power whose means of <manifestation|e x i s t e n c e   > has gone out of control,
- having lost its True Ancestor <coding|s h e l l   >.
+// Also maybe just reword this guy a bit
+It is no more than an incarnation of power whose means of <manifestation|e x i s t e n c e     > have gone out of control,
```

Comment:
With the loss of its True Ancestor shell, it is no more than an incarnation of power, whose means...